Dockerfile.client

```diff
@@ -75,9 +75,13 @@ COPY build build
 COPY src/clients src/clients
 COPY src/core src/core
 
+TRITON_COMMON_REPO_TAG=main
```

Comment:
You need to make these 

ARG TRITON_COMMON_REPO_TAG=main
ARG TRITON_CORE_REPO_TAG=main

at the top of the file, and then here have:

ARG TRITON_COMMON_REPO_TAG
ARG TRITON_CORE_REPO_TAG

Then in .gitlabci-yml you can use --build-arg to set them appropriately on the docker build invocation so that they get set to whatever is configuration for the CI run.

Reply:
Done.